openshift/pgweb-template.yaml

```diff
@@ -0,0 +1,122 @@
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: bayesian-pgweb
+    name: bayesian-pgweb
+  spec:
+    replicas: 1
+    selector:
+      service: bayesian-pgweb
+    template:
+      metadata:
+        labels:
+          service: bayesian-pgweb
+      spec:
+        containers:
+        - env:
+          - name: POSTGRESQL_DATABASE
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: database
+          - name: POSTGRESQL_INITIAL_DATABASE
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: initial-database
+          - name: POSTGRESQL_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: password
+          - name: POSTGRESQL_USER
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: username
+          - name: POSTGRES_SERVICE_HOST
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: host
+          - name: POSTGRES_SERVICE_PORT
+            valueFrom:
+              secretKeyRef:
+                name: coreapi-postgres
+                key: port
+          - name: PGBOUNCER_SERVICE_HOST
+            value: bayesian-pgbouncer
+          command: ["/usr/bin/pgweb"]
+          args: ["--url", "postgres://$(POSTGRESQL_USER):$(POSTGRESQL_PASSWORD)@$(PGBOUNCER_SERVICE_HOST)/$(POSTGRESQL_DATABASE)?sslmode=disable", "--bind=0.0.0.0", "--listen=8081"]
+          image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
+          name: bayesian-pgweb
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 8081
+            initialDelaySeconds: 10
+            periodSeconds: 60
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 8081
+            initialDelaySeconds: 10
+            timeoutSeconds: 5
+          ports:
+          - containerPort: 8081
+            protocol: TCP
+          resources:
+            requests:
+              memory: "128Mi"
+              cpu: "250m"
+            limits:
+              memory: "256Mi"
+              cpu: "500m"
+        restartPolicy: Always
+    test: false
+    triggers:
+    - type: ConfigChange
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      service: bayesian-pgweb
+    name: bayesian-pgweb
+  spec:
+    ports:
+    - name: "8081"
+      port: 8081
+      protocol: TCP
+      targetPort: 8081
+    selector:
+      service: bayesian-pgweb
+
+parameters:
+- description: Docker registry where the image is
+  displayName: Docker registry
+  required: true
+  name: DOCKER_REGISTRY
+  value: "docker.io"
+
+- description: Docker image to use
+  displayName: Docker image
+  required: true
+  name: DOCKER_IMAGE
+  value: "sosedoff/pgweb"
```

Comment:
It's better to rely on our own set of images. No issues at present, but we should use an image from quay.io or our own org under docker.io

Reply:
Since it is meant for debugging, I thought it is fine to pull it from outside. Is it fine to do it latter?

Reply:
Yes, that's absolutely fine.